src/cli.ts

```diff
@@ -61,7 +61,7 @@ function printHelp() {
   process.stdout.write(
 `
 ${pkg.name} ${pkg.version}
-Usage: json2ts [--input, -i] [IN_FILE] [--output, -o] [OUT_FILE]
+Usage: json2ts [--input, -i] [IN_FILE] [--output, -o] [OUT_FILE] [OPTIONS]
```

Comment:
Please add a line explaining what OPTIONS is (similar to what you did in the README)

Reply:
GitHub didn't close this review comment but the docs have actually been added to the help text below that line.